hw/ip/otbn/dv/uvm/otbn_sim_cfg.hjson

```diff
@@ -72,7 +72,7 @@ name:
 
   // TODO(lowrisc/opentitan#16689): Enable cdc instrumentation
   run_opts: ["+otbn_elf_dir={otbn_elf_dir}",
-             "+cdc_instrumentation_enabled=0"]
+             "+cdc_instrumentation_enabled=1"]
```

Comment:
nit: This change should be left out or put into a different commit ;)

Reply:
@matutem can you remove this change in this PR?

Reply:
Enabling otbn cdc was a slip. Removed from this PR.

Reply:
Thanks!